addons/account/static/src/js/reconciliation/reconciliation_renderer.js

```diff
@@ -109,8 +110,14 @@ var StatementRenderer = Widget.extend(FieldManagerMixin, {
                 'number': state.valuenow,
                 'timePerTransaction': Math.round(dt/1000/state.valuemax)
             }));
-            $done.appendTo(this.$el.first());
-            this.$('.o_automatic_reconciliation').hide();
+            this.$el.children().hide();
+            // display rainbowman after fully reconcilation
```

Comment:
remove line 114 -> 120, and replace them with 

this.trigger_up('rainbow_man', {fadeout: 'no', message: $done, click_close: false});